Server/src/assets/logo.png.license

```diff
@@ -0,0 +1,3 @@
+Copyright Contributors to the CitrineOS Project
```

Comment:
What's the correct license now? on all the other files I saw:
```
// Copyright (c) 2023 S44, LLC
// Copyright Contributors to the CitrineOS Project
//
// SPDX-License-Identifier: Apache 2.0
```

Reply:
the copyright from S44 is legacy, it shouldn't be removed but won't be used going forward. the SPDX header missing here is weird... the reuse tool made this file automatically when adding the headers. I'll need to check whether the tool is bugged or if there's a good reason the spdx header is missing.